package.json

```diff
@@ -4,6 +4,7 @@
 	"description": "Gitpod Support",
 	"publisher": "gitpod",
 	"version": "0.0.98",
+	"daemonVersion": "0.0.1",
```

Comment:
Do we really need 2 versions? How are we going to maintain it?

Reply:
That's because our daemon is not always updated when extension upgrade. I want to have notification to ask users to exit vscode desktop to get latest daemon features.

> Do we really need 2 versions?

But yep, we don't need it now

> How are we going to maintain it?

Original thought is to update this field after we change daemon

Reply:
> Original thought is to update this field after we change daemon

yes, but I'm not sure how ensure it. Anyway I think we are getting to far ahead of us. Let's land it somehow to try internally.